src/scss/objects/_form-toggle.scss

```diff
@@ -160,27 +139,40 @@ $formToggle-checked-text            : $green;
             background: $formToggle-background;
         }
 
-        .o-formToggle--mediumSized {
-            padding: $formToggle-padding ($formToggle-padding * 1.5);
-        }
+        .o-formToggle--withImage {
+            float: left;
+            overflow: hidden;
+            margin-bottom: spacing(x2);
+            width: calc(50% - #{spacing()});
+            padding: 63px $formToggle-padding ($formToggle-padding * 0.5);
 
-        .o-formToggle--mediumAboveMid {
-            @include media('>=mid') {
-                padding: $formToggle-padding ($formToggle-padding * 1.5);
+            &:nth-of-type(even) {
+                margin-left: spacing(x2);
             }
+
         }
 
-        .o-formToggle--largeSized {
-            height: $formToggle-large-size;
-            @include formToggle-large();
+        .o-formToggle--midInline {
+            @include media('<=mid') {
+                min-width: 150px;
+                margin: 0 spacing() 0 0;
+
+                &:nth-child(even) {
+                    margin-left: 0;
+                }
+            }
         }
 
-        .o-formToggle--largeAboveMid {
-            @include media('>=mid') {
-                float: left;
-                margin: 0 0 spacing();
-                @include formToggle-large();
+            .o-formToggle--withImage-container {
```

Comment:
I'd be tempted to structure this as:

```
.o-formToggle--withImage {
   & .o-formToggle-container {
   }
}
```

Just makes it a bit more future proof if other modifiers need to hook onto and use `o-formToggle-container`

Reply:
yeah, that makes sense - will do.